Can't print emojis on Windows

Currently I had to disable the CLI health check in the dev env GitHub Action because it [explodes](url) when trying to print our readme:

```
2022-08-05T16:48:16.5981518Z Traceback (most recent call last):
2022-08-05T16:48:16.5989130Z   File "C:\hostedtoolcache\windows\Python\3.10.5\x64\lib\runpy.py", line 196, in _run_module_as_main
2022-08-05T16:48:16.6072945Z     return _run_code(code, main_globals, None,
2022-08-05T16:48:16.6074114Z   File "C:\hostedtoolcache\windows\Python\3.10.5\x64\lib\runpy.py", line 86, in _run_code
2022-08-05T16:48:16.6074856Z     exec(code, run_globals)
2022-08-05T16:48:16.6075933Z   File "D:\a\hatch-fancy-pypi-readme\hatch-fancy-pypi-readme\src\hatch_fancy_pypi_readme\__main__.py", line 55, in <module>
2022-08-05T16:48:16.6076587Z     main()
2022-08-05T16:48:16.6077360Z   File "D:\a\hatch-fancy-pypi-readme\hatch-fancy-pypi-readme\src\hatch_fancy_pypi_readme\__main__.py", line 51, in main
2022-08-05T16:48:16.6077826Z     cli_run(cfg, out)
2022-08-05T16:48:16.6078517Z   File "D:\a\hatch-fancy-pypi-readme\hatch-fancy-pypi-readme\src\hatch_fancy_pypi_readme\_cli.py", line 50, in cli_run
2022-08-05T16:48:16.6079051Z     print(build_text(config.fragments), file=out)  # pragma: no cover
2022-08-05T16:48:16.6079572Z   File "C:\hostedtoolcache\windows\Python\3.10.5\x64\lib\encodings\cp1252.py", line 19, in encode
2022-08-05T16:48:16.6168727Z     return codecs.charmap_encode(input,self.errors,encoding_table)[0]
2022-08-05T16:48:16.6169952Z UnicodeEncodeError: 'charmap' codec can't encode character '\u2728' in position 45: character maps to <undefined>
2022-08-05T16:48:16.8289601Z ##[error]Process completed with exit code 1.
```

I have no clue if this is fixable at all?

@ofek you mentioned you use Windows? 😇

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -110,9 +110,9 @@ jobs:
       - uses: actions/setup-python@v3
         with:
           python-version: ${{env.PYTHON_LATEST}}
+      - run: chcp 65001
+        if: matrix.os == 'windows-latest'
       - run: python -m pip install -e .[dev]
       - run: python -c 'import hatch_fancy_pypi_readme'
       - run: python -m hatch_fancy_pypi_readme
-        if: matrix.os != 'windows-latest'
       - run: hatch-fancy-pypi-readme
-        if: matrix.os != 'windows-latest'

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -11,6 +11,7 @@ on:
 
 env:
   FORCE_COLOR: "1"  # Make tools pretty.
+  PYTHONIOENCODING: utf-8
   TOX_TESTENV_PASSENV: FORCE_COLOR
   PYTHON_LATEST: "3.10"
 
@@ -110,8 +111,6 @@ jobs:
       - uses: actions/setup-python@v3
         with:
           python-version: ${{env.PYTHON_LATEST}}
-      - run: chcp 65001
-        if: matrix.os == 'windows-latest'
       - run: python -m pip install -e .[dev]
       - run: python -c 'import hatch_fancy_pypi_readme'
       - run: python -m hatch_fancy_pypi_readme

